vhdbuilder/packer/components.json

```diff
@@ -180,7 +180,7 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/tigera/operator:*",
       "versions": [
-        "v1.13.5"
+        "v1.17.1"
```

Comment:
technically theres a bunch of otehr images this is brining in

Reply:
Should we add in all the Calico 3.19 images that this indirectly brings?

Reply:
thanks